### PR TITLE
Changes with subscriptions

### PR DESCRIPTION
**Pull request by anna-kartynnik**
### :scroll: Description

- Describe the general shape of this PR (new feature? refactor? bug fix? one-line change?)
New feature.

- Describe what changes are being made
`meeting_id` is removed from subscription creation function (from both frontend and backend). Also now users can subscribe to either text or email notifications or both of them.

- Describe why these changes are being made
Issues #193, #195, #218.

- List the use cases and edge cases relevant to this PR
Use subscribes to agenda items using phone, email or both of them.

- Describe how errors will be handled. How will we know if this code breaks in production
Errors from the server are shown on the page.

- Describe any external libraries/dependencies added or removed in this PR
n/a

- Describe any security risks are there regarding this change
n/a

- Describe how you tested these changes
Manually

- Link to relevant external documentation

--------------------------
### :clipboard: Mandatory Checklist

- [x] Linked to the Github Issues being addressed using the right sidebar :arrow_right:
- [x] Have you discussed these changes with the project leader(s)?
- [x] Do all variable and function names communicate what they do?
- [x] Were all the changes commented and / or documented?
- [x] Is the PR the right size? (If the PR is too large to review, it might be good to break it up into multiple PRs.)
- [x] Does all work in progress, temporary, or debugger code have a TODO comment with links to Github issues?
- [x] *If you changed the user interface, did you add before and after screenshots to below?*

--------------------------
#### After

https://user-images.githubusercontent.com/15210801/122870726-e583e400-d2e2-11eb-8874-f074ee6f7587.mov

--------------------------
### :blue_book: Glossary
- PR = Pull Request

### CODE REVIEW

File: frontend/src/components/Subscribe/Subscribe.jsx [hunk 1]
```diff
@@ -38,23 +30,35 @@ import { convertQueryStringToServerFormat } from './subscribeQueryString';
...
   const [phoneError, setPhoneError] = useState(null);
   const [emailError, setEmailError] = useState(null);
+  const [subscriptions, setSubscriptions] = useState(null);
+
+  const [createSubscriptions, { loading, error }] = useMutation(
+    CREATE_SUBSCRIPTIONS,
+    {
+      onCompleted: data => data && data.createSubscriptions ?
+        setSubscriptions(data.createSubscriptions) : setSubscriptions(null)
+    }
```
Comment (terchiem) [inline frontend/src/components/Subscribe/Subscribe.jsx:60]: Can be simplified to:
```suggestion
    {
      onCompleted: data => setSubscriptions(data?.createSubscriptions ?? null)
    }
```

**Review by terchiem — APPROVED** [review]
Nice work 👍

File: frontend/src/components/Subscribe/Subscribe.jsx (hunk 1 above)
Comment (anna-kartynnik) [reply frontend/src/components/Subscribe/Subscribe.jsx]: Oh, thank you @terchiem, it's nice. I never used this operator before. Has it been added recently or did I just miss it?

Comment (terchiem) [reply frontend/src/components/Subscribe/Subscribe.jsx]: It's pretty new, I think it was added last year.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

**Review by JMStudiosJoe — APPROVED** [review]

**Pull request merged**